docusaurus/docs/publish-a-plugin/build-automation.md

```diff
@@ -18,42 +18,171 @@ keywords:
+          attestation: true
+```
+
+:::note
+It is likely this file already exists in your repository. Verify its content and make any necessary changes to match the example.
+:::
+
+## How to trigger the release workflow
+
+To trigger the release workflow you need to push a tag with the format `vX.X.X` to the repository. Typically all of your changes will be merged into `main`, and the tag is applied to `main`
+
+### Create a `vX.X.X` tag with your package manager (Recommended)
+
+The easiest way to create a version tag is using your package manager.
+
+In the following examples a patch version (following [Semantic Versioning](https://semver.org/)) is created:
+
+with [npm](https://docs.npmjs.com/cli/v7/commands/npm-init):
+
+```
+npm version patch
+```
 
-## Create a release tag
+with [yarn](https://yarnpkg.com/lang/en/docs/cli/version/):
+
+```
+yarn version patch
+```
+
+with [pnpm](https://pnpm.io/):
+
+```
+pnpm version patch
+```
+
+This updates your version in the `package.json` file and creates a new Git tag with the format `vX.X.X`. You can change `patch` to `minor` or `major` if you want to create a new minor or major version.
+
+After creating the tag, push it to the repository:
+
+```bash
+git push origin main --tags
+```
 
-A tag with the format `vX.X.X` is used to trigger the release workflow. Typically all of your changes will be merged into `main`, and the tag is applied to `main`
+<details>
+  <summary>Alternatively, create a `vX.X.X` tag manually</summary>
+
+### Create a `vX.X.X` tag manually
+
+If you prefer, you can also create the tag manually using the following commands:
 
 ```BASH
 git checkout main
 git pull origin main
-git tag v2.0.1
-git push origin v2.0.1
+git tag v2.0.1 # adjust the version accordingly
+git push origin main --tags
 ```
 
+### Retag a release
+
 If you need to re-tag the release, the current tag can be removed with these commands:
 
 ```BASH
-git tag -d v2.0.1
+git tag -d v2.0.1 # adjust the version accordingly
 git push --delete origin v2.0.1
 git checkout main
 git pull origin main
 ```
 
 After you push the tag, you can create the same tag again.
 
+Once you have created the tag, you can push it to the repository:
+
+```bash
+# assuming your default branch is `main`
+git push origin main --tags
+```
+
+</details>
+
+## Publish your release in Github
+
+After you [create and push the tag](#how-to-trigger-the-release-workflow), the release workflow will run, generating a release with all the artifacts needed to submit your plugin to the [Grafana plugin catalog](https://grafana.com/plugins).
+
+The workflow creates draft releases. You can edit the release in GitHub, update the description as needed, and then publish it. For more details on managing repository releases, refer to the [GitHub documentation](https://docs.github.com/en/repositories/releasing-projects-on-github/managing-releases-in-a-repository).
```

Comment:
```suggestion
The workflow creates a **draft release**. You can edit the release in GitHub, update the description as needed, and then publish it. For more details on managing repository releases, refer to the [GitHub documentation](https://docs.github.com/en/repositories/releasing-projects-on-github/managing-releases-in-a-repository).
```
I feel like its important to indicate it is draft. Also should we advise on making it a true release _after_ you have received feedback from the review?

Reply:
@academo any thoughts on:
> Also should we advise on making it a true release after you have received feedback from the review?

Reply:
@sympatheticmoose in the docs we mention `Once the draft release is published, you can use the release assets to submit your plugin to the [Grafana plugin catalog](https://grafana.com/plugins). Simply copy the links to the archive (zip) file and sha1 sum. Use these in the plugin submission form.`

I want to propose we start creating published releases in the automation, since you have to commit a tag of a version to get the automation to trigger,. there's clear intention that you want to publish the release.